python/lsst/pipe/analysis/utils.py

```diff
@@ -1249,18 +1254,22 @@ def getRepoInfo(dataRef, coaddName=None, coaddDataset=None, doApplyUberCal=False
     metaStr = coaddName + coaddDataset + "_md" if coaddName else "calexp_md"
     metadata = butler.get(metaStr, dataId)
     hscRun = checkHscStack(metadata)
-    dataset = "src"
+    catDataset = "src"
     skymap = butler.get(coaddName + "Coadd_skyMap") if coaddName else None
     wcs = None
     tractInfo = None
     if isCoadd:
         coaddImageName = "Coadd_calexp_hsc" if hscRun else "Coadd_calexp"  # To get the coadd's WCS
         coadd = butler.get(coaddName + coaddImageName, dataId)
         wcs = coadd.getWcs()
-        dataset = coaddName + coaddDataset
+        catDataset = coaddName + coaddDataset
         tractInfo = skymap[dataId["tract"]]
-    if doApplyUberCal:
-        dataset = "wcs_hsc" if hscRun else "jointcal_wcs"
+    photoCalibDataset = None
+    if doApplyExternalPhotoCalib:
+        photoCalibDataset = "fcr_hsc" if hscRun else externalPhotoCalibName + "_photoCalib"
```

Comment:
This is for meas_mosaic, right?  Is there a reason this isn't deprecated?  (Would it still run, even?) And, what does `hscRun` mean?  If this is meas_mosaic related, then maybe this should be renamed `measMosaicRun` or something?

Reply:
`meas_mosaic` is indeed effectively deprecated.  I haven't yet stripped out accommodations for `meas_mosaic` calibrations as runs still exist in the wild and "just in case" (there is still the occasional murmur about having to run it on the HSC calls...this would be very sad, but why spend time ripping out the functionality until all murmurs subside?).   As for `hscRun`, this actually means "was run using the HSC-stack", i.e. the forked version of many years ago.  This has long been retired, and ripping out this functionality is on the to-do list of https://jira.lsstcorp.org/browse/DM-14559 (but low-priority as it "does no harm").

>     hscRun : `str` or `None`, optional
>        String representing "HSCPIPE_VERSION" fits header if the data were
>        processed with the (now obsolete, but old reruns still exist)
>        "HSC stack".  Default is `None`.
>     matchRadius : `float` or `None`, optional
>